cetmix_tower_server/models/cx_tower_server.py

```diff
@@ -1574,40 +1577,37 @@ def _prepare_ssh_command(self, command_code, path=None, sudo=None, **kwargs):
             command (list|str): command splitted into separate commands
                 (for sudo with password mode ('p')) or composed command
                 from its parts for sudo without password mode ('n'))
-
         """
-        # Prepare command for sudo if needed
         if sudo:
-            # Add location
             sudo_prefix = "sudo -S -p ''"
+            no_split = kwargs.get("no_split_for_sudo", False)
 
-            # Detect command separator
-            separator = "&&"
-            if separator in command_code:
-                result = (
-                    command_code.replace("\\", "").replace("\n", "").split(separator)
-                )
-
-                # Sudo with password expects a list of commands
-                result = [f"{sudo_prefix} {cmd.strip()}" for cmd in result]
-
-                # Merge back into a single command is sudo is without password
-                if sudo == "n":
-                    result = f" {separator} ".join(result)
-            else:
-                # Single command only
+            if no_split:
```

Comment:
Once again: we don't need to add any extra condition. We need to modify the existing one.